app/models/audit_log.rb

```diff
@@ -89,7 +89,7 @@ def self.create_audit_record(user)
         tenant: user.tenant,
         actor: user,
         actor_name: user.name
-      )
+      ) if user
```

Comment:
Toto kedy moze nastat?

Reply:
Neviem, na to som neprisla (nesedi mi to, ze by user mohol byt nil, lebo audit record sa robi ked sa user prihlasi alebo odhlasi), ale vychadzame z rollbar errorov, nastal taky pripad.